sdk/tests/conformance2/textures/misc/tex-mipmap-levels.html

```diff
@@ -189,6 +211,30 @@
       wtu.glErrorShouldBe(gl, gl.NO_ERROR, "generateMipmap should succeed");
   }
   gl.deleteTexture(tex);
+
+  // Test textureSize should work correctly with non-zero base level
+  var program = wtu.setupProgram(
+      gl, ['vshader_texsize', 'fshader_texsize'], ['vPosition'], [0]);
+
+  gl.uniform1i(gl.getUniformLocation(program, "tex"), 0);
+  gl.uniform4f(gl.getUniformLocation(program, "uMult"), 1, 1, 1, 1);
+  gl.uniform1i(gl.getUniformLocation(program, "lod"), 1);
+  gl.uniform2i(gl.getUniformLocation(program, "texSize"), 7, 4);
+  tex = gl.createTexture();
+  gl.bindTexture(gl.TEXTURE_2D, tex);
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texParameter(TEXTURE_MAG_FILTER) should succeed");
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.LINEAR);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texParameter(TEXTURE_MIN_FILTER) should succeed");
+  gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_BASE_LEVEL, 1);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "texParameter(TEXTURE_BASE_LEVEL) should succeed");
+  gl.texStorage2D(gl.TEXTURE_2D, 4, gl.RGBA8, 31, 17);
```

Comment:
Is there a reason to use these odd numbers? Why not use a size that at least is evenly divisible by 4 in both dimensions?